Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Applications/WindowsApp/Locators/CalculatorWindow.cs

```diff
@@ -4,23 +4,23 @@
 namespace Aquality.Selenium.Core.Tests.Applications.WindowsApp.Locators
 {
     public static class CalculatorWindow
-    {
-        public static By WindowLocator => By.TagName("Window");
+    { 
+        public static By WindowLocator => MobileBy.ClassName("ThunderRT6FormDC");
 
-        public static By OneButton => By.Name("1");
+        public static By OneButton => By.XPath("//*[@Name='1']");
```

Comment:
By.Name locator strategy is supported according to documentation https://appium.io/docs/en/drivers/windows/, why did you removed it?
Have you tried [MobileBy.Name](https://github.com/appium/appium-dotnet-driver/pull/477/files#diff-932485d2a1783d91e3a63922a9a94e411a659988ec4a703bdbe342d0911879c8)? Or is it a breaking change of Selenium 4 that affects our FindElements implementation, as it was in java?

Reply:
By.Name removed due to tests failures. Example: ```OpenQA.Selenium.NoSuchElementException : Element [+] was not found by locator 'By.Name: +'```.
It is possible to use MobileBy.Name. I've checked and its is working fine. Will be updated

Reply:
updated